### ISSUE TITLE
There is no zcashd-wallet-tool in Docker image

### ISSUE THREAD

**Issue by taoeffect** [zcash/zcash#5895]
### Describe the issue

There's no `zcashd-wallet-tool` in 4.7.0 docker image.

#### If so, please list the steps to reproduce below:

1. `docker pull electriccoinco/zcashd:v4.7.0`
2. run a container and `exec` into it with `bash`
3. Run `zcashd-wallet-tool`

### Expected behaviour

Should run.

### Actual behaviour + errors

Command not found.

### The version of Zcash you were using:

`Zcash Daemon version v4.7.0-gitian`

### Machine specs:

- Linux

**Comment by charlieok** [comment]
This was a result of zcashd-wallet-tool being left out of our 4.7.0 debian package. The 4.7.0 docker image was built using that package so it has that issue as well.

We addressed this by building a 4.7.0+1 debian package. I've now built, tagged, and published a docker image tagged v4.7.0-1 using that newer debian package.

Thank you for the report!

**Comment by daira** [comment]
I verified that zcashd-wallet-tool is in the v4.7.0-1 image:
```
daira@katava:~/docker$ docker images |grep v4.7.0-1
electriccoinco/zcashd                  v4.7.0-1   4fcd830e8328   23 hours ago    201MB
daira@katava:~/docker$ docker run -it --entrypoint /bin/bash 4fcd830e8328
zcashd@73b62c1fac08:~$ ls -l /usr/bin/zcashd-wallet-tool
-rwxr-xr-x 1 root root 2042496 Apr 18 18:11 /usr/bin/zcashd-wallet-tool
```
